app/notifiers/session_reminder_notifier.rb

```diff
@@ -28,7 +28,7 @@ def title
       end
     end
 
-    def subject
+    def mail_title
       if params[:time_before_session].match?(/^0\s/)
         t("mailers.session_mailer.reminder.subject.without_time")
```

Comment:
Need to update the structure in the `en.yml` file as these are no longer subject related strings